### PR TITLE
Fix dependant bot PR bump stripe-android from 10.4.3 to 11.0.5 #2342

### PR DESCRIPTION
**Pull request by VMadalin**
### Description
Fixed dependentBot PR https://github.com/fossasia/open-event-attendee-android/pull/2342 related with this issue https://github.com/fossasia/open-event-attendee-android/issues/2341

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt [hunk 1]
```diff
@@ -871,14 +871,15 @@ class AttendeeFragment : Fragment(), ComplexBackPressFragment {
     }
 
     private fun sendToken(card: Card) {
-        Stripe(requireContext())
-            .createToken(card, BuildConfig.STRIPE_API_KEY, object : TokenCallback {
-                override fun onSuccess(token: Token) {
-                    val charge = Charge(attendeeViewModel.getId().toInt(), token.id, null)
+        Stripe(requireContext(), BuildConfig.STRIPE_API_KEY)
+            .createToken(card, object : ApiResultCallback<Token> {
+                override fun onSuccess(result: Token) {
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt]: Revert to token

File: app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt [hunk 2]
```diff
@@ -871,14 +871,15 @@ class AttendeeFragment : Fragment(), ComplexBackPressFragment {
...
+                    val charge = Charge(attendeeViewModel.getId().toInt(), result.id, null)
                     attendeeViewModel.chargeOrder(charge)
                 }
-                override fun onError(error: Exception) {
-                    rootView.snackbar(error.localizedMessage.toString())
+
+                override fun onError(e: Exception) {
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt]: Revert to error

**Comment by rob729** [comment]
@VMadalin I am already working on this issue #2341

**Comment by liveHarshit** [comment]
@VMadalin You should have asked before working on it?

File: app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt (hunk 1 above)
Comment (VMadalin) [reply app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt]: The interface has been changes with this new one: 
```java
public interface ApiResultCallback<ResultType> {
    void onSuccess(@NonNull ResultType result);
    void onError(@NonNull Exception e);
}
```
Normally following code conventions we should keep the implementing members in the same order as members of the interface, and keeping the methods properties names. Imagine the implementation of this method `void onError(@NonNull Exception e);` with the following variations `e`, `error`, `exception` ...

But it's not problem to change :), if you prefer

**Comment by VMadalin** [comment]
@rob729, @liveHarshit My apologise for create directly the PR , but  I saw the tag  `help wanted` without any PR associated. And due theses changes are very small, I push directly

File: app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt (hunk 1 above)
Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt]: > Normally following code conventions we should keep the implementing members in the same order as members of the interface, and keeping the methods properties names.

If that was actually true, Java wouldn't have allowed to rename or reorder the methods.

Suppose I write an interface
```kotlin
interface Result<T> {
  onSuccess(taekwondo: T)

  onError(elephant: Error)
}
```

Should the calling code use `taekwondo` and `elephant` as variable names or more appropriately named `token` and `error`

Comment (VMadalin) [reply app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt]: I'm absolutely agree with you but , for me are two different things one thing is code-style and another compiler. For example `javascript` give you the option to implement a solution in different ways without warnings but that doesn't mean that all are well.

Please consult the following link: https://kotlinlang.org/docs/reference/coding-conventions.html#interface-implementation-layout  that have reference to this.

**Comment by liveHarshit** [comment]
> @rob729, @liveHarshit My apologise for create directly the PR , but I saw the tag `help wanted` without any PR associated. And due theses changes are very small, I push directly

It's okay. Please take care of it by next time,
Thank you.

**Pull request closed without merging**